packages/web/src/components/track/DownloadSection.tsx

```diff
@@ -308,6 +316,25 @@ export const DownloadSection = ({ trackId }: DownloadSectionProps) => {
                 isOriginal={quality === DownloadQuality.ORIGINAL}
               />
             ))}
+            {uploadingStems.map((s, i) => (
```

Comment:
how does the ux feel here after the upload finishes? can play with it on stage after merge

Reply:
had an issue with confirmer, maybe stage was regressed a bit when i was testing. otherwise wouldve done a video. but yeah let's play with it